### ISSUE TITLE
accessibility and folding

### ISSUE THREAD

**Issue by webczat** [microsoft/vscode#92933]
From what I know, folding is automatically disabled when accessibility is enabled.
I am wondering, why is so? As in, Except some weird eclipse bug that made me sometimes type things into places where I was not supposed to be typing, I was successfully using folding with a screenreader to be able to quickly skim through source with ability to still read things like source comments. Somehow it is more natural for me sometimes than switching to outline or invoking a symbol list. It may be useful to consider whether it isn't a bad idea to make it configurable for us too.

**Comment by alexdima** [comment]
@webczat Folding is not automatically disabled when accessibility is enabled. Are you on the latest version of VS Code?

**Comment by webczat** [comment]
I have read somewhere it is (although I may be incorrect) and, I haven't seen it influence anything. I am on latest 1.44 insiders. I can check if I didn't also disable it, though.

**Comment by webczat** [comment]
I indeed did disable folding. On the other hand I really don't see it influenced my editing when I re-enabled it.

**Comment by alexdima** [comment]
@webczat Thank you for following up. Most likely, the documentation is out of date. If you find the place which is out of date, please let us know... This used to be the case perhaps a year ago.

**Comment by webczat** [comment]
well, as said, I haven't seen any folding being applied here after I enabled it... not sure what I am doing wrong

**Comment by alexdima** [comment]
@webczat Can you please try the following:
* create a new text file with the content:
```
function f() {
	return 5;
}
```
* Then move the cursor to the beginning of the file (folding actions will not fold a region if it contains the cursor).
* Then run `F1 > Fold All`
* Now the body of the function should be folded.

Does that work for you?

**Comment by webczat** [comment]
seems to work, tested on my own code. Although I don't see anything being folded by default, but that is likely another thing.

**Comment by alexdima** [comment]
@webczat Thank you for following up. Regardless of accessibility turned on or off, VS Code does not fold anything by default AFAIK. I will close this issue for now since there is nothing to be done.
